svr-runopts.c

```diff
@@ -459,6 +470,13 @@ void svr_getopts(int argc, char ** argv) {
 		}
 	}
 #endif
+
+#if DROPBEAR_RESTRICT_FIXED_HOST_IP
```

Comment:
don't turn this mandatory. Remove this block.

Reply:
Done.